icon_validator/rules/workflow_validators/workflow_use_case_validator.py

```diff
@@ -0,0 +1,77 @@
+import requests
+from icon_plugin_spec.plugin_spec import KomandPluginSpec
+
+from icon_validator.rules.validator import KomandPluginValidator
+from icon_validator.exceptions import ValidationException
+
+
+class WorkflowUseCaseValidator(KomandPluginValidator):
+    @staticmethod
+    def get_approved_usecase_tags(response_json: dict) -> [str]:
+        approved_usecases = []
+        for result in response_json["results"]:
+            if result["type"] == "usecase":
+                approved_usecases.append(result["name"])
+
+        return approved_usecases
+
+    @staticmethod
+    def get_approved_usecase_tags_with_paging() -> [str]:
+        approved_usecases = []
+        query = "types=usecase"
+        for i in range(0, 9999):
+            response = requests.get(url=f"https://extensions-api.rapid7.com/v2/public/tags?first=1000&{query}")
+            response_json = response.json()
+            approved_usecases.extend(WorkflowUseCaseValidator.get_approved_usecase_tags(response_json))
+            if not response_json["pageInfo"]["hasNextPage"]:
+                break
+
+            query = f"after={response_json['pageInfo']['endCursor']}"
+
+        return approved_usecases
+
+    @staticmethod
+    def validate_use_case_exists(spec: KomandPluginSpec):
+        try:
+            use_cases = spec.spec_dictionary()["hub_tags"]["use_cases"]
+            if not use_cases or not len(use_cases):
```

Comment:
```suggestion
            if not use_cases:
```
You can consolidate them as `if not var` checks for False, None, and 0 length